pkg/addons/data/ServiceAccount/openshift-branding/branding.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
should not need to specify a ServiceAccount at all - by default "default" is used and that's fine